test/functional/tests/security/test_load_corrupted_metadata.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# Copyright(c) 2023-2024 Huawei Technologies Co., Ltd.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import random
+
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CacheLineSize,
+    CleaningPolicy,
+    PromotionPolicy,
+    SeqCutOffPolicy,
+)
+from storage_devices.device import Device
+from core.test_run import TestRun
+from storage_devices.ramdisk import RamDisk
+from test_tools.dd import Dd
+from test_utils.output import CmdException
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.parametrizex("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrizex("promotion_policy", PromotionPolicy)
+@pytest.mark.parametrizex("seq_cutoff_policy", SeqCutOffPolicy)
+def test_load_corrupted_metadata(cache_mode, cache_line_size, cleaning_policy,
+                                 promotion_policy, seq_cutoff_policy):
+    """
+        title: Security test for loading cache with corrupted metadata.
+        description: |
+          Validate the ability of Open CAS to load cache instance and work if metadata is corrupted.
```

Comment:
`            Validate the ability of Open CAS to load cache instance and work if metadata is corrupted.`